### ISSUE TITLE
what's maven dependency to use

### ISSUE THREAD

**Issue by yongqxu** [stephenh/dtonator#12]
do i need to add maven dependency to use this tool in Intellij? i couldn't find it from maven central.

**Comment by stephenh** [comment]
Hey; yeah, I never got around to publishing this to Maven central; I've got a small Maven repo at `repo.joist.ws` with `dtonator` in it, or you could just jitpack to pull it in directly from the github project:

https://jitpack.io/

Although I don't have a top-level build.gradle directory; it's nested inside of a sub directory, so I'm not sure how/if that would work; I've not done it before.

If you figure it out, let me know and I'll add it do the docs. Thanks!

**Comment by yongqxu** [comment]
i AWS S3 repo to save the jars, do you have detail instructions of how to setup in Intellij IDE and get it
to work?

**Comment by stephenh** [comment]
No; I used Eclipse at the time, although I've been using IntelliJ lately. Happy to accept documentation improvements if you get a nice setup working.
